app/src/app/initApp.js

```diff
@@ -54,14 +54,21 @@ export default () => {
     height: 1000
   })
 
+  // figure out if we start in background
+  const autostart = (process.argv || []).indexOf('--autostart') !== -1
```

Comment:
process.argv is always an array, why do you need `(process.argv || [])`

Reply:
copied that from a github comment 😇
will change